pom.xml

```diff
@@ -122,6 +147,26 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.xolstice.maven.plugins</groupId>
+                <artifactId>protobuf-maven-plugin</artifactId>
+                <version>${protobuf-maven-plugin.version}</version>
+                <configuration>
+                    <protocArtifact>com.google.protobuf:protoc:3.5.1-1:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
do you really need this `${os.detected.classifier}`?

Reply:
yes. this way is how the plugin get the `os classifier` as could be inconsistencies between JVM and OS.